bigtable/client/client_options_test.cc

```diff
@@ -14,6 +14,15 @@
 
 #include "bigtable/client/client_options.h"
 
+#include <cstdlib>
+#ifdef WIN32
```

Comment:
Here and elsewhere - _WIN32 is preferable (leading underscore variant) versus WIN32.